### PR TITLE
small enhancement to dnsutil module

### PR DESCRIPTION
**Pull request by The-Loeki**
- Modify `A` to use non-deprecated C function
- Add `AAAA` function to do the same as the `A` function
- minor documentation & typo fixes

### CODE REVIEW

**Comment by jfindlay** [comment]
@The-Loeki, thanks for the pull request.  I'm hesitant to merge new code into a stable release branch, but I'll leave that up to @thatch45.

File: salt/modules/dnsutil.py [hunk 1]
```diff
@@ -245,17 +245,42 @@ def A(host, nameserver=None):
...
+
+    return 'This function requires dig, which is not currently available'
+
+
+def AAAA(host, nameserver=None):
+    '''
+    Return the AAAA record(s) for `host`.
+
+    Always returns a list.
+
```
Comment (jfindlay) [inline salt/modules/dnsutil.py]: This needs a `.. versionadded:: 2014.7.5` here.

Comment (The-Loeki) [inline salt/modules/dnsutil.py]: There you go :+1: 

We'll see; If deemed more appropriate, I'll PR against 2015.2.

**Comment by thatch45** [comment]
Yes, normally this should go onto the develop branch, but I will let this in

**Pull request merged**